### PR TITLE
fix: remove older tav versions

### PR DESCRIPTION
**Pull request by astorm**
Fixes: #2172 

This PR removes older versions of Apollo Server from the TAV test matrix in order to make the test cases compatible for both apollo-server-express@3.x and apollo-server-express@2.x

The apollo-server-express@3.x tests require us to call `apollo.start` to start the apollo server. 

However, the some older version of apollo-server-express@2.x  (< 2.8.2) don't have a `start` method.

This PR removed these over two years old versions of apollo server from the test matrix in order to get a clean run.  Other options include

1. Breaking off separate TAV suites for apollo < 2.8.2
2.Rewriting the apollo tests to detect versions and use `listen` or `start` accordingly to start the server. 

### Checklist

- [x] Implement code

### CODE REVIEW

**Comment by astorm** [comment]
@trentm Re:

>  If not, then I'd bias to still testing 2.x (perhaps not back to 2.0).

Agreed.  The version we're cutting off at (2.8.2) was released almost two years ago (23 months and some change)

```
% npm view apollo-server-express time | grep 2.8.2
  '2.8.2': '2019-08-20T18:03:00.440Z',
```

File: .tav.yml [hunk 1]
```diff
@@ -221,7 +221,7 @@ apollo-server-express-2_12:
   # but only the latest MAJOR.MINOR.x to reduce the test matrix.
```
Comment (trentm) [inline .tav.yml:221]: nit: can we update the "We want this version range: ..." comment to say why we are clipping at >2.8?

**Review by trentm — APPROVED** [review]

File: .tav.yml (hunk 1 above)
Comment (trentm) [reply .tav.yml:221]: meh, nevermind. Encoded in the PR discussion here.

**Comment by trentm** [comment]
Some context for:

> If not, then I'd bias to still testing 2.x (perhaps not back to 2.0).

This was about proposing to allow dropping TAV tests of versions older than 18months, somewhat in line with https://www.elastic.co/support/eol policy.

File: .tav.yml (hunk 1 above)
Comment (astorm) [reply .tav.yml:221]: Update the versions in the comments, thanks for catching that.

File: package.json [hunk 2]
```diff
@@ -137,7 +137,6 @@
     "dependency-check": "^4.1.0",
     "elasticsearch": "^16.5.0",
     "express": "^4.17.1",
-    "express-graphql": "^0.9.0",
```
Comment (trentm) [inline package.json:140]: Was this unintentional? Won't that break:

```
test/config.test.js
715:  var expressGraphqlVersion = require('express-graphql/package.json').version
727:    modules.delete('express-graphql')

test/instrumentation/modules/express-graphql.test.js
17:var graphqlHTTP = require('express-graphql')
```

?

**Review by trentm — APPROVED** [review]

**Pull request merged**